templates/CRM/Contribute/Form/Contribution.tpl

```diff
@@ -241,52 +241,58 @@
   </table>
 
   {if !$contributionMode}
-    <fieldset class="payment-details_group">
-      <legend>
-        {ts}Payment Details{/ts}
-      </legend>
-        <table class="form-layout-compressed" >
-          <tr class="crm-contribution-form-block-payment_instrument_id">
-            <td class="label">{$form.payment_instrument_id.label}</td>
-            <td {$valueStyle}>{$form.payment_instrument_id.html} {help id="payment_instrument_id"}</td>
-            </td>
-          </tr>
-          {if $showCheckNumber || !$isOnline}
-            <tr id="checkNumber" class="crm-contribution-form-block-check_number">
-              <td class="label">{$form.check_number.label}</td>
-              <td>{$form.check_number.html}</td>
-            </tr>
-          {/if}
-          <tr class="crm-contribution-form-block-trxn_id">
-            <td class="label">{$form.trxn_id.label}</td>
-            <td {$valueStyle}>{$form.trxn_id.html} {help id="id-trans_id"}</td>
-          </tr>
-          {if $email and $outBound_option != 2}
-            <tr class="crm-contribution-form-block-is_email_receipt">
-              <td class="label">
-                {$form.is_email_receipt.label}</td><td>{$form.is_email_receipt.html}&nbsp;
-                <span class="description">{ts 1=$email}Automatically email a receipt for this payment to %1?{/ts}</span>
+      <div class="crm-accordion-wrapper crm-accordion_title-accordion crm-accordion-processed payment-details_group">
+        <div class="crm-accordion-header">
+          {ts}Payment Details{/ts}
+        </div>
+        <div class="crm-accordion-body">
+        {if $contribID && $payments}
```

Comment:
Shouldn't we just determine if we are in edit mode - perhaps use Core_Action - seems like this is a round-about-way of saying that. I guess you think it should be suppressed if there are no payments because it might seem 'obvious' that there aren't any